app/presenters/two_factor_options_presenter.rb

```diff
@@ -56,6 +65,10 @@ def intro
     end
   end
 
+  def show_cancel_phishing_resistant_only
```

Comment:
We'll want to make sure this behavior applies for both PIV-only and phishing-resistant-only flows.

Reply:
Yes. I suspected that was too specific of an approach. Thanks!